core/src/main/scala/org/apache/spark/deploy/history/FsHistoryProvider.scala

```diff
@@ -487,8 +487,10 @@ private[history] class FsHistoryProvider(conf: SparkConf, clock: Clock)
   protected def mergeApplicationListing(fileStatus: FileStatus): Unit = {
     val eventsFilter: ReplayEventsFilter = { eventString =>
       eventString.startsWith(APPL_START_EVENT_PREFIX) ||
-        eventString.startsWith(APPL_END_EVENT_PREFIX) ||
-        eventString.startsWith(LOG_START_EVENT_PREFIX)
+      eventString.startsWith(APPL_END_EVENT_PREFIX) ||
+      eventString.startsWith(LOG_START_EVENT_PREFIX) ||
+      eventString.startsWith(JOB_START_EVENT_PREFIX) ||
+      eventString.startsWith(JOB_END_EVENT_PREFIX)
```

Comment:
Do you know if adding these adds any significant time/delay to page loads?

Reply:
Actually i have not do any benchmark test for this modify.But it has been tested with our production cluster.